examples/todos/android/src/main/java/org/reduxkotlin/example/todos/TodoAdapter.kt

```diff
@@ -0,0 +1,46 @@
+package org.reduxkotlin.example.todos
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import kotlinx.android.synthetic.main.item_todo.view.*
+import org.reduxkotlin.examples.todos.Todo
+import org.reduxkotlin.examples.todos.ToggleTodo
+import android.graphics.Paint.STRIKE_THRU_TEXT_FLAG
+
+
+
+//TODO - animate changes - adding/removing/strikeout
+class TodoAdapter : RecyclerView.Adapter<TodoViewHolder>() {
```

Comment:
If you extend `ListAdapter<T, ViewHolder>`, the changes will automatically be animated.

Reply:
💯 done